src/patternfly/components/Wizard/wizard.scss

```diff
@@ -8,6 +8,11 @@
   --pf-c-wizard--lg--MaxHeight: calc(100% - (var(--pf-global--spacer--2xl) * 2));
   --pf-c-wizard--m-full-width--lg--MaxWidth: auto;
   --pf-c-wizard--m-full-height--lg--Height: 100%;
+  --pf-c-wizard--m-in-page--BoxShadow: none;
```

Comment:
I can't completely remember, but didn't we decide that if you want to style a component in another component for one specific use case, then you should style that component in the component that has the new variation. Similar to the case of the notification icon in the page header... instead of making a variation of the button that has a background we style the button/icon in the page component. Maybe I'm wrong, but that seems like a similar case here where you should make the variation of the wizard inside the page. Seems like thats the only place where this variation will occur?

Reply:
Oh this variation basically removes the wizard from being constrained to a width/height for use in it's own version of a modal. This variation can be used in the page, in a `pf-c-modal-box` or whatever, and will eventually just be the default wizard (in a breaking change release).

Reply:
I left a comment related to this here, specifically around the modifier name. https://github.com/patternfly/patternfly-next/pull/2186#discussion_r318142645